Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx

```diff
@@ -142,12 +142,15 @@ int main(int argc, char *argv[])
       }
     }
   // write out the copy
-  vtkNew<vtkMRMLMarkupsFiducialStorageNode> outputFiducialStorageNode;
-  outputFiducialStorageNode->SetFileName(seedsOutFile.c_str());
-  // the .xml file specifies that it expects the output file in LPS
-  // coordinate system
-  outputFiducialStorageNode->UseLPSOn();
-  outputFiducialStorageNode->WriteData(copiedFiducialNode.GetPointer());
+  if (!seedsOutFile.empty())
+    {
+    vtkNew<vtkMRMLMarkupsFiducialStorageNode> outputFiducialStorageNode;
+    outputFiducialStorageNode->SetFileName(seedsOutFile.c_str());
+    // the .xml file specifies that it expects the output file in LPS
+    // coordinate system
+    outputFiducialStorageNode->UseLPSOn();
```

Comment:
This is independent of this PR but I am wondering if there is way to get the value associated with the `coordinateSystem` attribute:

https://github.com/Slicer/Slicer/blob/8db9e8e573b73d82384db97633697bda5aee4401/Modules/CLI/ExecutionModelTour/ExecutionModelTour.xml#L206-L212